src/gui/SplashScreen.C

```diff
@@ -370,7 +373,7 @@ SplashScreen::SplashScreen(bool cyclePictures) : QFrame(0, Qt::SplashScreen)
            << tr("October")
            << tr("November")
            << tr("December");
-    int currentMonth = 7;
+    int currentMonth = 12;
```

Comment:
Sorry, thought I added this comment with the previous.
Are you sure we will be getting this out in December, with holidays and vacations coming up?